Cargo.toml

```diff
@@ -50,48 +47,46 @@ std = [
   "futures-channel",
   "async-mutex",
 ]
-alloc = [
-  "futures-core/alloc",
-  "pin-project-lite",
-]
+alloc = ["futures-core/alloc", "pin-project-lite"]
 tokio02 = ["smol/tokio02"]
 
 [dependencies]
 async-attributes = { version = "1.1.1", optional = true }
 async-task = { version = "3.0.0", optional = true }
-async-mutex = { version = "1.1.3", optional = true }
+async-mutex = { version = "1.1.5", optional = true }
 crossbeam-utils = { version = "0.7.2", optional = true }
-futures-core = { version = "0.3.4", optional = true, default-features = false }
-futures-io = { version = "0.3.4", optional = true }
+futures-core = { version = "0.3.5", optional = true, default-features = false }
+futures-io = { version = "0.3.5", optional = true }
 kv-log-macro = { version = "1.0.6", optional = true }
 log = { version = "0.4.8", features = ["kv_unstable"], optional = true }
 memchr = { version = "2.3.3", optional = true }
-num_cpus = { version = "1.12.0", optional = true }
-once_cell = { version = "1.3.1", optional = true }
-pin-project-lite = { version = "0.1.4", optional = true }
-pin-utils = { version = "0.1.0-alpha.4", optional = true }
+num_cpus = { version = "1.13.0", optional = true }
+once_cell = { version = "1.4.0", optional = true }
+pin-project-lite = { version = "0.1.7", optional = true }
+pin-utils = { version = "0.1", optional = true }
```

Comment:
There are a number of changes of the versions which are compatible versions. I recently learned about `--skip-compatible` option of `cargo upgrade` and started to use this. It seems like a good way to reduce unnecessary code changes.